kbs/docker/kbs-client-image/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM docker.io/library/rust:1.80.0 AS builder
+
+WORKDIR /usr/src/kbs
+COPY . .
+
+RUN apt-get update && apt-get install -y pkg-config libssl-dev git sudo
```

Comment:
cc @fitzthum @bpradipt 

This is something related to https://github.com/confidential-containers/trustee/pull/764#discussion_r2043754190

Note that in this PR the image to be built is "sample-only" for x86-64/s390x/aarch64